app/src/main/java/org/p2p/wallet/alarmlogger/model/AlarmSwapErrorConverter.kt

```diff
@@ -3,18 +3,18 @@ package org.p2p.wallet.alarmlogger.model
 import com.google.gson.Gson
 import org.p2p.wallet.alarmlogger.api.AlarmErrorsRequest
 import org.p2p.wallet.alarmlogger.api.AlarmErrorsSwapRequest
-import org.p2p.wallet.home.repository.UserTokensRepository
 import org.p2p.core.network.data.ServerException
 import org.p2p.core.utils.toJsonObject
 import org.p2p.wallet.jupiter.repository.transaction.JupiterSwapTransactionMapper
 import org.p2p.wallet.jupiter.statemanager.SwapProfiler
 import org.p2p.wallet.jupiter.statemanager.SwapState
 import org.p2p.core.crypto.Base58String
+import org.p2p.wallet.user.repository.UserTokensLocalRepository
 
 class AlarmSwapErrorConverter(
     private val gson: Gson,
     private val transactionMapper: JupiterSwapTransactionMapper,
-    private val userTokensRepository: UserTokensRepository,
+    private val userTokensRepository: UserTokensLocalRepository,
```

Comment:
i am not sure that it's good for architecture to have dependency on repository impl

it's better if we just inject UserTokensRepository and inside there is logic to understand where it should fetch data from.

in current case we need to remember that we need to inject only local repository and ask ourself questions "should i inject remote repository or.. local repostiory? which is working?" everytime we need user tokens

Reply:
As I've discovered, UserTokensLocalRepository is in fact an interface, not an implementation, the naming is somewhat confusing. It would be better to maintain the name UserTokensRepository, as we don't have a UserTokensRemoteRepository to serve as a counterpart